fe/fe-core/src/main/java/com/starrocks/catalog/IcebergResource.java

```diff
@@ -60,7 +62,11 @@ protected void setProperties(Map<String, String> properties) throws DdlException
 
         catalogType = properties.get(ICEBERG_CATALOG);
         if (StringUtils.isBlank(catalogType)) {
-            throw new DdlException(ICEBERG_CATALOG + " must be set in properties");
+            catalogType = properties.get(ICEBERG_CATALOG_LEGACY);
+            if (StringUtils.isBlank(catalogType)) {
+                throw new DdlException(ICEBERG_CATALOG + " must be set in properties");
+            }
+            LOG.warn(ICEBERG_CATALOG_LEGACY + " will be deprecated, using " + ICEBERG_CATALOG + " instead.");
```

Comment:
This can be removed because users can't see this log.

Reply:
Okay, I'll also revise the Chinese version of docs after this PR is merged.